crates/bitwarden-exporters/src/cxf/export.rs

```diff
@@ -46,41 +45,32 @@ impl TryFrom<Cipher> for Item {
         let mut credentials: Vec<Credential> = value.r#type.clone().into();
 
         if let Some(note) = value.notes {
-            credentials.push(Credential::Note(Box::new(NoteCredential { content: note })));
+            credentials.push(Credential::Note(Box::new(NoteCredential {
+                content: note.into(),
+            })));
         }
 
         Ok(Self {
             id: value.id.as_bytes().as_slice().into(),
             creation_at: Some(value.creation_date.timestamp() as u64),
             modified_at: Some(value.revision_date.timestamp() as u64),
-            ty: value.r#type.try_into()?,
             title: value.name,
             subtitle: None,
             favorite: Some(value.favorite),
             credentials,
             tags: None,
             extensions: None,
+            scope: match value.r#type {
+                CipherType::Login(login) => Some((*login).into()),
+                _ => Some(CredentialScope {
+                    urls: vec![],
```

Comment:
If this is a valid scope, we could consider having CredentialScope implement default in the `credential_exchange_types` crate

Reply:
This was intended to be None but `CredentialScope` was required for a bit and I forgot to update it.